experiments/multi/slotnick_etal_2001/slotnick_figures_234.m

```diff
@@ -0,0 +1,94 @@
+% Recreate figures 3 and 4.
+matlabpool open 4
+
+% Add paths
```

Comment:
@vishaalprasad This is the code I added, to create figures 2-4. Can you have a looksie and leave any comments/